internal/clients/stubs/gcp_stub.go

```diff
@@ -51,6 +51,10 @@ func getServiceGCPClientStub(ctx context.Context) (si clients.ServiceGCP, err er
 	return si, err
 }
 
+func (moc *GCPClientStub) GetServiceAccountPrincipal(ctx context.Context) clients.GcpPrincipal {
+	return clients.GcpPrincipal("serviceAccount:provisioning-stage@red-hat-hcc.iam.gserviceaccount.com")
```

Comment:
Not a huge fan of hardcoding it, don't we have the information in the GCP JSON? if it would not be hard to get it from there, I'd very much prefer that, but if we can't do it easily, hardcoding it is fine :)

Reply:
Done

Reply:
gotcha, might be better not to include real data in stub, but not a big deal :)